src/tools/pub/glossary.md

```diff
@@ -146,16 +146,34 @@ and C are transitive ones.
 
 ## Uploader
 
-An uploader of a package is someone who has administrative permissions
-for that package. They can not only upload new versions of a package,
-but also [add and remove other uploaders](/tools/pub/cmd/pub-uploader)
-for that package. The uploader of a package is often, but not necessarily, the
-same as the [author](/tools/pub/pubspec#authorauthors) of a package.
+Someone who has administrative permissions for a package.
+A package uploader can upload new versions of the package,
+and they can also [add and remove other uploaders](/tools/pub/cmd/pub-uploader)
+for that package. 
+
+If a package has a verified publisher,
+then all members of the publisher can upload the package.
 
 Anyone uploading a new package automatically becomes an uploader for
-that package. Otherwise, to become an uploader, you need to contact an
+that package. Otherwise, to become an uploader of a package that
+doesn't have a verified publisher, you need to contact an
 existing uploader and ask them to add you as another uploader.
 
+
+## Verified publisher
```

Comment:
Here, rather than repeating all the details of what a verified publisher is, consider focusing on what members of a publisher have the rights to do (like in the previous Uploaders section). That is:

  - Publish new packages under the publisher
  - Publish new versions of existing packages
  - Edit the publisher metadata

Reply:
I just stole this definition from pub.dev, since it seemed worthwhile to have the definition on this site too. I didn't want to get into functionality here, since it's easy to miss updating the glossary. Instead I just linked to the verified publishers page. (We don't have any similar page for uploaders, so I guess it makes sense to have the info here.)

(I just checked, and people actually do come to the glossary, yay.)

Reply:
Even if this is the glossary, why the asymmetry between this section (which talks about publishers from a consumers PoV) and the Uploader section (which talks about uploaders from a publisher PoV)?

Reply:
Funny, I don't see the audiences of the entries as being that different; it's only the degree of completeness that's different, because there's no page dedicated to the uploader concept. The only link to glossary#uploader is from /tools/pub/cmd/pub-uploader, which is intended for publishers; there are no links to glossary#verified-publisher because there's a verified publisher page.

I'm going to cut down the verified publisher entry, since it's only here for completeness.

Reply:
I've made the glossary changes (and some others while I'm in there).

If we want to say anything more about publishers, we should say it in one of the other docs, since it's unlikely people will come here to find out about verified publishers.

We might also want to revamp the verified publishers page to be clearer about which sections are for which audiences.